packages/components/src/palette-edit/index.tsx

```diff
@@ -535,6 +538,8 @@ export function PaletteEdit( {
 											) }
 											{ canReset && (
 												<Button
+													__next40pxDefaultSize
+													className="components-palette-edit__menu-button"
```

Comment:
The last menu item is visible when the `canReset` prop is enabled. It was also missing a CSS class that set the width to 100%.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/c74d91c8-5f8d-4762-a710-2abccba4415c" alt="PaletteEdit dropdown, before" width="216">|<img src="https://github.com/user-attachments/assets/f5498392-092e-4a13-aaf5-d5e901ba8fcc" alt="PaletteEdit dropdown, after" width="216">|

Reply:
Right, it looks like this could be a DropdownMenu.

@ciampo Should we add any new dropdown instances using DropdownMenuV2, or should we still use v1 until it's a bit more stable?

Reply:
Let's keep v1 for now, but use this as a place where we can use v2 soon (probably as soon as https://github.com/WordPress/gutenberg/pull/66383 is merged)

Reply:
Related to https://github.com/WordPress/gutenberg/issues/61094